### PR TITLE
Stakater platform

### PR DESCRIPTION
**Pull request by aliartiza75**

### CODE REVIEW

**Comment by stakater-user** [comment]
@aliartiza75 Yikes! You better fix it before anyone else finds out! [Build 1](https://jenkins.release.stakater.com/job/stakater/job/Playbook/job/PR-70/1/) has Failed!

**Comment by stakater-user** [comment]
@aliartiza75 Image is available for testing. `docker pull docker.delivery.stakater.com:443/stakater/playbook:SNAPSHOT-PR-70-2`

**Comment by stakater-user** [comment]
@aliartiza75 Image is available for testing. `docker pull docker.delivery.stakater.com:443/stakater/playbook:SNAPSHOT-PR-70-3`

**Comment by stakater-user** [comment]
@aliartiza75 Image is available for testing. `docker pull docker.delivery.stakater.com:443/stakater/playbook:SNAPSHOT-PR-70-4`

**Comment by stakater-user** [comment]
@aliartiza75 Image is available for testing. `docker pull docker.delivery.stakater.com:443/stakater/playbook:SNAPSHOT-PR-70-5`

**Comment by stakater-user** [comment]
@aliartiza75 Image is available for testing. `docker pull docker.delivery.stakater.com:443/stakater/playbook:SNAPSHOT-PR-70-6`

**Comment by stakater-user** [comment]
@aliartiza75 Image is available for testing. `docker pull docker.delivery.stakater.com:443/stakater/playbook:SNAPSHOT-PR-70-7`

**Comment by stakater-user** [comment]
@aliartiza75 Image is available for testing. `docker pull docker.delivery.stakater.com:443/stakater/playbook:SNAPSHOT-PR-70-8`

**Comment by stakater-user** [comment]
@aliartiza75 Image is available for testing. `docker pull docker.delivery.stakater.com:443/stakater/playbook:SNAPSHOT-PR-70-9`

**Comment by stakater-user** [comment]
@aliartiza75 Image is available for testing. `docker pull docker.delivery.stakater.com:443/stakater/playbook:SNAPSHOT-PR-70-10`

**Comment by stakater-user** [comment]
@aliartiza75 Yikes! You better fix it before anyone else finds out! [Build 11](https://jenkins.release.stakater.com/job/stakater/job/Playbook/job/PR-70/11/) has Failed!

**Comment by stakater-user** [comment]
@aliartiza75 Image is available for testing. `docker pull docker.delivery.stakater.com:443/stakater/playbook:SNAPSHOT-PR-70-12`

**Comment by stakater-user** [comment]
@aliartiza75 Image is available for testing. `docker pull docker.delivery.stakater.com:443/stakater/playbook:SNAPSHOT-PR-70-13`

**Comment by stakater-user** [comment]
@aliartiza75 Image is available for testing. `docker pull docker.delivery.stakater.com:443/stakater/playbook:SNAPSHOT-PR-70-14`

**Comment by stakater-user** [comment]
@aliartiza75 Image is available for testing. `docker pull docker.delivery.stakater.com:443/stakater/playbook:SNAPSHOT-PR-70-18`

File: content/stacks/stakaterplatform.md [hunk 1]
```diff
@@ -12,149 +12,33 @@ Stakater Platform consist of 7 stacks;
 - [Alerting](https://playbook.stakater.com/content/stacks/alerting.html)
 - [Tracing](https://playbook.stakater.com/content/stacks/tracing.html)
 
-## Platfrom Deployment
+## Why StakaterPlatform ?
 
-This section contains steps to deploy Stakater Platform on Kubernetes cluster using Gitlab CI pipeline.
+StakaterPlatform gives you a head start for managing your kubernetes cluster by providing open source tools across six stacks so that you only concern about developing and releasing applications on Kubernetes.
```
Comment (ahmedwaleedmalik) [inline content/stacks/stakaterplatform.md:17]: seven stacks

**Review by ahmedwaleedmalik — APPROVED** [review]

**Comment by stakater-user** [comment]
@aliartiza75 Image is available for testing. `docker pull docker.delivery.stakater.com:443/stakater/playbook:SNAPSHOT-PR-70-19`

**Pull request merged**